pkg/exchange/coinbase/stream_handlers.go

```diff
@@ -153,6 +153,38 @@ func (s *Stream) buildMapForUserDataStream(channel2LocalSymbolsMap map[types.Cha
 	// - order life cycle events: receive, open, done, change, activate(for stop orders)
 	// - order match
 	channel2LocalSymbolsMap[userChannel] = privateChannelLocalSymbols
+
+	// query account id for the symbols
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*20)
+	defer cancel()
+	markets, err := s.exchange.QueryMarkets(ctx)
+	if err != nil {
+		logStream.WithError(err).Warn("fail to query markets")
```

Comment:
you can add a FieldLogger i. the stream struct for reuse

Reply:
done.